src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/vedtak/PeriodeService.kt

```diff
@@ -19,12 +19,25 @@ class PeriodeService(
 ) {
     fun erEnsligForsørgerOgPerioderLike(behandlingId: UUID): SkalSammenslåPerioder {
         val fagsak = fagsakRepository.finnFagsakForBehandlingId(behandlingId)
+        if (harKunEnPeriode(behandlingId)) return SkalSammenslåPerioder.IKKE_AKTUELT
+
         if (erPerioderLike(behandlingId) && fagsak.ytelsestype.tilTema() == Tema.ENF) {
             return SkalSammenslåPerioder.JA
         }
+
         return SkalSammenslåPerioder.IKKE_AKTUELT
     }
 
+    private fun harKunEnPeriode(behandlingId: UUID): Boolean {
```

Comment:
Mulig jeg surrer, men jeg tenker egentlig at "motsatt" er mer riktig;
hvis man har flere perioder i både fakta, vilkår og foreldelse - da er det aktuelt å vurdere sammenslåing..?

Eller tenker jeg feil?

At saksbehandler splitter en vilkårsvurderingsperiode som var sammenhengende gjør jo ikke at denne koden er aktuell å vurdere, egentlig? De vil isåfall gi litt "false-positive" på "NEI"-alternativet, siden er ment å være ulike.

Reply:
Slik vi har lagd det er det mulig å sammenslå perioder selv om det bare er en periode til å starte med, men hvor man splitter vilkårsvurdering. Hvis begge vilkårsvurderingene er helt like blir resultatet to like avsnitt i brev, som gjør at det er aktuelt å sammenslå?

Reply:
Det er dessverre slik at vi baserer funksjonalitet på denne verdien. Det vil si at dersom de periodene som er splittet i vilkårsvurdering, men som er likt utfylt, vil ikke kunne sammenslås dersom de regnes som ikke aktuelle. I utgangspunktet skal de ikke trenge å splitte disse periodene uansett, så kanskje det er greit?

Reply:
Konkludert sammen med Charlie at det er greit at dersom det bare finnes en faktaperiode som splittes og som er likt utfylt ikke blir sammenslått.